helm-chart/sefaria-project/templates/cronjob/reindex-elasticsearch-es6.yaml

```diff
@@ -0,0 +1,77 @@
+{{- if .Values.cronJobs.reindexElasticSearch.enabled }}
```

Comment:
maybe set the to cronjobs.reindexElasticSearch6 and  cronjobs.reindexElasticSearch8